release_notes.md

```diff
@@ -4,4 +4,6 @@
 -->
 
 **Release sprint:** Sprint 88
+- Update Python Worker to 1.1.8 [Release Note](https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.8)
```

Comment:
This is for sprint88 but we want this to be added to Sprit 86/87.
Release notes were dropped here:
https://github.com/Azure/azure-functions-host/commit/8cd9be240a36de50cfef5961533dbafb6786eb2a